vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -200,7 +200,8 @@ $global:map = @{
     "c:\akse-cache\win-vnet-cni\" = @(
         "https://acs-mirror.azureedge.net/azure-cni/v1.4.22/binaries/azure-vnet-cni-singletenancy-windows-amd64-v1.4.22.zip",
         "https://acs-mirror.azureedge.net/azure-cni/v1.4.32/binaries/azure-vnet-cni-singletenancy-windows-amd64-v1.4.32.zip",
-        "https://acs-mirror.azureedge.net/azure-cni/v1.4.32/binaries/azure-vnet-cni-singletenancy-swift-windows-amd64-v1.4.32.zip"
```

Comment:
FYI: Remove the cache when you make sure customers will not use it after 2022-10B (2022-10-11)